lib/clamp-device.in

```diff
@@ -29,7 +29,9 @@ KMOPTOPT="${KMOPTOPT:="-O3"}"
 # pass extra options to LLC
 # KMOPTLLC can be used to pass last-minute options to llc in the backend
 # if not set, then "-O2" will be passed to llc
-KMOPTLLC="${KMOPTLLC:="-O2"}"
+# add "--frame-pointer=none" as deafult option to llc to enable frame
+# pointer elimination, so saved registers can be used for SGPR spill
+KMOPTLLC="${KMOPTLLC:="-O2 --frame-pointer=none"}"
```

Comment:
I think the -O2  here is a bug (not introduced by this PR) since it will get overwritten by the "-O3" in $HCC_OPT below, could you remove it?